### ISSUE TITLE
SVG images massed up

### ISSUE THREAD

**Issue by marmor7** [andrieslouw/imagesweserv#96]
Great service, thanks!

I've found that it doesn't work well for SVG images:

e.g.:
http://www.contactspls.com/files/theme/images/cplus_logo.svg
<img width="1235" alt="original" src="https://user-images.githubusercontent.com/1893538/27009123-c44038ac-4e8e-11e7-9adf-3aa2a8010086.png">

https://images.weserv.nl/?url=www.contactspls.com%2Ffiles%2Ftheme%2Fimages%2Fcplus_logo.svg&h=200&w=200
<img width="257" alt="result" src="https://user-images.githubusercontent.com/1893538/27009125-c9bbf848-4e8e-11e7-9d1f-95e3fd678ab0.png">

**Comment by kleisauke** [comment]
Most SVG images are outputted correctly. However, some SVG images have problems including this one (this is usually due to limitations of `librsvg` that we use to load SVG images). When I tried to open this SVG in `GIMP` it has the same (incorrect-)output as you've specified (in `Inkscape` it was correct).

I couldn't find exactly the root cause why this one has render problems, but it looks like it was incorrectly optimized (the path rounding/numbering is incorrect and CSS is used instead of SVG attributes).

I've tried to fix your SVG image (uploaded [here](https://t0.nl/cplus-logo.svg)).

Output:
[![image](https://user-images.githubusercontent.com/12746591/27010213-771c2782-4e9f-11e7-8bad-e06fd4b80e50.png)](https://images.weserv.nl/?url=t0.nl/cplus-logo.svg&h=200&w=200)
